src/proposals/20230802/SparkEthereum_20230802.t.sol

```diff
@@ -303,6 +303,84 @@ contract SparkEthereum_20230802Test is SparkTestBase, TestWithExecutor {
+
+        // Ensure WETH liquidity
+        _deposit(weth, POOL, lp, 1_000_000e18);
+
+        // Setup some positions ahead of time that can later be liquidated
+        // Position is 350 ETH so ~700k
+        _deposit(dai, POOL, user2, 1_000_000e18);
+        this._borrow(weth, POOL, user2, 350e18, false);
+
+        _deposit(dai, POOL, user3, 1_000_000e18);
+        _deposit(weth, POOL, user3, 1_000e18);
+        this._borrow(weth, POOL, user3, 1_000e18, false);
+
+        _executePayload(address(payload));
+
+        // --- Test 1 - Cannot Borrow More than small amount against DAI ---
+
+        // Refresh the configs just in case
+        configs = createConfigurationSnapshot('', POOL);
+        dai = _findReserveConfig(configs, DAI);
+        weth = _findReserveConfig(configs, WETH);
+
+        // Deposit 1m
+        _deposit(dai, POOL, user1, 1_000_000e18);
+
+        // Should only be able to borrow small amounts of ETH
+        // 1m * 1% = $10k = ~5 ETH (assume price of 2k / ETH)
+        this._borrow(weth, POOL, user1, 1e18, false);
+
+        // Cannot borrow 10
+        vm.expectRevert(bytes('36'));	// COLLATERAL_CANNOT_COVER_NEW_BORROW
+        this._borrow(weth, POOL, user1, 10e18, false);
+
+        // --- Test 2 - Can liquidate any single position that was previously setup ---
+
+        // Liquidate the position setup previously
+        assertEq(IERC20(dai.underlying).balanceOf(liquidator1), 0);
+        assertEq(IERC20(dai.aToken).balanceOf(user2), 1_000_000e18);
+
+        _liquidate(dai, weth, POOL, liquidator1, user2, 350e18);
+
+        // Liquidator should get about 700k DAI (depends on actual price of ETH)
+        assertApproxEqAbs(IERC20(dai.underlying).balanceOf(liquidator1), 700_000e18, 50_000e18);
+
+        // User can keep remainder
+        assertApproxEqAbs(IERC20(dai.aToken).balanceOf(user2), 300_000e18, 50_000e18);
+
+        // --- Test 3 - Liquidate multi-collateralized position ---
+
+        // We can fully liquidate the DAI position which now contributes almost nothing to HF
+        assertEq(IERC20(dai.underlying).balanceOf(liquidator2), 0);
+        assertEq(IERC20(weth.underlying).balanceOf(liquidator2), 0);
+        assertEq(IERC20(dai.aToken).balanceOf(user3), 1_000_000e18);
+        assertEq(IERC20(weth.aToken).balanceOf(user3), 1_000e18);
+        
+        // Can only liquidate about half the debt, but this will make the position healthy
+        // Can only do half because there is only 1m DAI collateral for 2m in debt
+        _liquidate(dai, weth, POOL, liquidator2, user3, 1_000e18);
+
+        assertApproxEqAbs(IERC20(dai.underlying).balanceOf(liquidator2), 1_000_000e18, 50_000e18);
+        // Some WETH is leftover because the liquidation call was limited by the amount of DAI available
+        assertApproxEqAbs(IERC20(weth.underlying).balanceOf(liquidator2), 500e18, 50e18);
+        assertApproxEqAbs(IERC20(dai.aToken).balanceOf(user3), 0, 1);
```

Comment:
```suggestion
        assertApproxEqAbs(IERC20(dai.underlying).balanceOf(liquidator2), 1_000_000e18, 50_000e18);

        // Some WETH is leftover because the liquidation call was limited by the amount of DAI available
        assertApproxEqAbs(IERC20(weth.underlying).balanceOf(liquidator2), 500e18, 50e18);
        assertApproxEqAbs(IERC20(dai.aToken).balanceOf(user3),            0,      1);
```